src/OpenSSL/crypto.py

```diff
@@ -2025,6 +2030,70 @@ def export(self, cert, key, type=FILETYPE_PEM, days=100,
 
 
 class PKCS7(object):
+
+    def __init__(self, pkey=None, cert=None, cacerts=None, pkcs7=None):
+        self._pkey = pkey
+        self._cert = cert
+        self._cacerts = cacerts
+        self._pkcs7 = pkcs7
+
+    def get_certificate(self):
```

Comment:
It is better to return/support list of certificates, because pkcs7 could contain multiple certs.
Furthermore openssl API provides following methods:
- PKCS7_get0_signers() - to get stack of certs
- sk_X509_num() - to get amount of certs
- sk_X509_value() - to get cert from stack